src/types/Hashed.ts

```diff
@@ -0,0 +1,29 @@
+import * as bcryptC from 'bcrypt'
+import * as Promise from 'bluebird'
+import * as _ from 'lodash'
+
+const bcrypt:any = Promise.promisifyAll(bcryptC)
```

Comment:
We're only using 3 methods from it. If you pomisify them individually  you'll get the proper type signatures gratis